### PR TITLE
Frontend: Fix sorting of organization popup in alphabetical order

### LINKED ISSUE #22162: Switch organization popup not order alphabetically
**What would you like to be added**:
In the popup to switch to an org, the organization should be alphabetically ordered (like it was before)

**Why is this needed**:
I have like a thousand org, trying to find the one is ... hard

### PR DESCRIPTION
**Pull request by vikkyomkar**
**What this PR does / why we need it**:
organization switch popup list was sorted by OrgId, this PR sort the organizations alphabetically

**Which issue(s) this PR fixes**:
Fixes #22162 

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by torkelo** [comment]
Think it’s better if this was already sorted by the http api (and in sql query)

**Comment by vikkyomkar** [comment]
@torkelo changes done as per the suggestion

**Comment by vikkyomkar** [comment]
@torkelo @aknuds1 can you please review this change ?
This is similar as #22349

**Comment by vikkyomkar** [comment]
Hi @torkelo  @aknuds1 This PR is pending from long time.
Can you please review it ?

File: pkg/services/sqlstore/user.go [hunk 1]
```diff
@@ -317,7 +317,7 @@ func GetUserOrgList(query *models.GetUserOrgListQuery) error {
 	sess.Join("INNER", "org", "org_user.org_id=org.id")
 	sess.Where("org_user.user_id=?", query.UserId)
 	sess.Cols("org.name", "org_user.role", "org_user.org_id")
-	sess.OrderBy("org.name")
+	sess.OrderBy("LOWER(org.name),org.name")
```
Comment (aknuds1) [inline pkg/services/sqlstore/user.go:320]: Can you please revert this change? After discussing the matter internally, we've decided that it has too much of a performance impact to lowercase org.name as part of the query.

**Review by aknuds1 — CHANGES_REQUESTED** [review]

**Comment by aknuds1** [comment]
Also please merge with latest Grafana master, in order to fix the CI issues.

**Review by aknuds1 — APPROVED** [review]
LGTM

**Comment by aknuds1** [comment]
We discussed how to best solve this internally, and reached the compromise that we make the ordering (case insensitive primarily, case sensitive secondarily) in the API. Hope you don't mind I pushed these changes to your branch, in order to expedite the PR.

**Review by marefr — APPROVED** [review]
LGTM

**Review by papagian — APPROVED** [review]
LGTM!

**Comment by aknuds1** [comment]
Sorry about the long time it took to resolve the PR, but now it's merged. Thanks for contributing to Grafana!

**Pull request merged**